examples/examples_py_test.go

```diff
@@ -21,6 +21,15 @@ func TestSimplePython(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestUntypedPython(t *testing.T) {
+	test := getPythonBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: filepath.Join(getCwd(t), "untyped-py"),
+		})
+
+	integration.ProgramTest(t, &test)
```

Comment:
I suggest we extend this test to exercise Create, Update, Refresh, and Delete for the resource, so that we cover all operations. Also, test what's written in the `outputs` stack output.

Reply:
All the operations are covered by the standard program test. Yes, asserting on outputs would be nice - but the test failed hard at the first step when I originally wrote it so the first milestone was just getting it to not crash during the basic operations 😅